EasyPost/Services/CarrierAccountService.cs

```diff
@@ -58,5 +66,25 @@ public async Task<CarrierAccount> Retrieve(string id)
         }
 
         #endregion
+
+        private static string GenerateEndpoint(string carrierAccountType)
+        {
+            var registerEndpointCarriers = new List<string>
+            {
+                "UpsAccount",
+                "FedexAccount"
+            };
+
+            string endpoint = "carrier_accounts";
```

Comment:
This seems like an antipattern to define a default twice. Let's either set it here above or let the switch case take care of this.

Reply:
Resolved